lib/oneview-sdk/rest.rb

```diff
@@ -121,6 +123,75 @@ def rest_delete(path, options = {}, api_ver = @api_version)
       rest_api(:delete, path, options, api_ver)
     end
 
+    # Uploads a file to a specific uri
+    # @param [String] file_path
+    # @param [String] uri The uri starting with "/"
+    # @param [Hash] body_params The params to append to body of http request. Default is {}.
+    # @option body_params [String] 'name' The name to show (when resource accepts a name)
+    # @param [Integer] timeout The number of seconds to wait for completing the request. Default is 300.
+    # @return [OneviewSDK::Resource] if the upload was sucessful, return a Resource object
```

Comment:
Does this actually return a Resource? It looks like it returns the response_handler return value (a response body)